src/app/malaria-101/activity-3/activity-3.component.ts

```diff
@@ -122,8 +124,14 @@ export class OddOneOutComponent implements OnInit {
         this._questionNumber++;
         if (this._questionNumber === 5) {
             this.activityComplete = true;
-          if (!this.completed) { this._performanceService.openDialog(5); }
-          this._sharedData.customAlert(this.language.alerts.title, this.solutions, 'info');
+            swal({
+              title: this.language.alerts.title,
+              html: this.solutions,
+              type: 'info',
+            }).then( res => { if (!this.completed) {
+            this._badgeService.updateBadgeNumber(2).subscribe(response => response);
```

Comment:
instead of using the number 2 and 5 and these constants create constants in the file.

like const stageTwo = 2;  then use them.

Reply:
Done.